libr/cons/cons.c

```diff
@@ -635,6 +635,14 @@ R_API int r_cons_eof() {
 	return feof (I.fdin);
 }
 
+R_API void r_cons_go_forward (int x) {
+	printf ("\x1b[%dC", x);	
```

Comment:
uhm.. not sure how is this  going  to be used

Reply:
Yea, it is useless. I was trying out a different approach initially and forgot to undo it. I will remove it.